docs/zh-CN/docs/lakehouse/multi-catalog/jdbc.md

```diff
@@ -164,6 +164,32 @@ JDBC Catalog 通过标准 JDBC 协议，连接其他数据源。
 > 
 > 3. Http 地址。如：`https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/jdbc_driver/mysql-connector-java-5.1.47.jar`。系统会从这个 http 地址下载 Driver 文件。仅支持无认证的 http 服务。
 
+## 数据查询
+
+```sql
+select * from mysql_table where k1 > 1000 and k3 ='term';
+```
+由于可能存在使用数据库内部的关键字作为字段名，为解决这种状况下仍能正确查询，所以在SQL语句中，会根据各个数据库的标准自动在字段名与表名上加上转义符。例如 MYSQL(``)、PostgreSQL("")、SQLServer([])、ORACLE("")，所以此时可能会造成字段名的大小写敏感，具体可以通过explain sql，查看转义后下发到各个数据库的查询语句。
+
+## 数据写入
+
```

Comment:
wrap with `<version since="dev"> </version>`
See: https://doris.apache.org/zh-CN/community/how-to-contribute/contribute-doc#%E6%96%87%E6%A1%A3%E5%A4%9A%E7%89%88%E6%9C%AC

Reply:
And add english doc too.

Reply:
done